pkg/api/grpc/api.go

```diff
@@ -70,22 +71,23 @@ type API interface {
 	// Methods internal to the object
 	SetAppChannel(appChannel channel.AppChannel)
 	SetDirectMessaging(directMessaging messaging.DirectMessaging)
-	SetActorRuntime(actor actors.Actors)
 }
 
 type api struct {
-	*universalapi.UniversalAPI
+	*universal.Universal
```

Comment:
Isn't this still stuttering?

Reply:
Having the same struct name as the package name is generally ok. We remove `API` because this is implied from the context of being in the `pkg/api` package path.